src/missed_spans.rs

```diff
@@ -34,6 +34,12 @@ impl<'a> FmtVisitor<'a> {
         })
     }
 
+    pub fn format_missing_comments(&mut self, end: BytePos) {
+        self.format_missing_inner(end, |_, _, _| {
+            // do nothing
```

Comment:
Because this is called where only comments or white spaces exist. I updated to do `this.buffer.push_str(last_snippet.trim_right());` since I just wanted a version of `format_missing_indent` that doesn't insert indents.